Sources/Apollo/GraphQLGETTransformer.swift

```diff
@@ -43,6 +43,8 @@ public struct GraphQLGETTransformer {
     }
 
     components.queryItems = queryItems
+    components.percentEncodedQuery =
+      components.percentEncodedQuery?.replacingOccurrences(of: "+", with: "%2B")
```

Comment:
I wonder if this needs to actually be replaced in the query items rather than in the percent encoded query for it to essentially get re-percent-encoded so it's not read by the server as a raw string.